File: VB/Console/SingleInheritance.vb
class A
	Dim a as Integer
	Sub New()
		a=10
	End Sub
	public Sub showa()
		Console.WriteLine("a=" & a)
	End Sub
End Class

class B 
	Inherits A
	Private b as Integer
	public Sub New()
		b=20
	End Sub
	public Sub New(x as Integer)
		b = x
	End Sub
	public Sub showb()
		Console.WriteLine("b=" & b)
	End Sub
End Class

Module SingleInheritance
	Sub Main()
		Dim obb as new B(30)
	
		obb.showa()
		obb.showb()
	End Sub
End Module

File: CS/Console/SingleInheritance.cs
using System;

class A
{
	private int a;
	public A()
	{
		a=10;
	}
	public void showa()
	{
		Console.WriteLine("a=" + a);
	}
}

class B : A
{
	private int b;
	public B()
	{
		b=20;
	}
	public void showb()
	{
		Console.WriteLine("b=" + b);
	}
}

class SingleInheritance
{
	public static void Main(string[] args)
	{
		B obb = new B();
	
		obb.showa();
		obb.showb();
	}
}